### PR TITLE
More shuffleboard layout improvements

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

File: shuffleboard.json [hunk 1]
```diff
@@ -184,24 +170,80 @@
...
-              1
+              3,
+              2
             ],
             "content": {
-              "_type": "Text View",
+              "_type": "Graph",
               "_source0": "network_table:///SmartDashboard/Drivetrain Left Position",
-              "_title": "Encoder Left"
+              "_title": "/SmartDashboard/Drivetrain Left Position",
```
Comment (andrewda) [inline shuffleboard.json]: Sir

File: shuffleboard.json [hunk 2]
```diff
@@ -184,24 +170,80 @@
...
-              1,
-              1
+              3,
+              2
             ],
             "content": {
-              "_type": "Text View",
+              "_type": "Graph",
+              "_source0": "network_table:///SmartDashboard/Drivetrain Right Position",
+              "_title": "/SmartDashboard/Drivetrain Right Position",
```
Comment (andrewda) [inline shuffleboard.json]: Ugh why did you change all the names back

File: shuffleboard.json [hunk 3]
```diff
@@ -184,24 +170,80 @@
...
+          "3,3": {
+            "size": [
+              3,
+              2
+            ],
+            "content": {
+              "_type": "Graph",
               "_source0": "network_table:///SmartDashboard/Drivetrain Right Position",
-              "_title": "Encoder Right"
+              "_title": "/SmartDashboard/Drivetrain Right Position",
```
Comment (andrewda) [inline shuffleboard.json]: DAMMIT SAVEAU

File: shuffleboard.json [hunk 4]
```diff
@@ -184,24 +170,80 @@
...
+          },
+          "6,0": {
+            "size": [
+              3,
+              2
+            ],
+            "content": {
+              "_type": "Graph",
+              "_source0": "network_table:///SmartDashboard/Drivetrain Pitch",
+              "_title": "/SmartDashboard/Drivetrain Pitch",
```
Comment (andrewda) [inline shuffleboard.json]: I hope you fix your mistakes this took me so long to do

**Comment by SUPERCILEX** [comment]
Lol, rebased.
